src/org/parosproxy/paros/network/HtmlParameter.java

```diff
@@ -26,7 +26,7 @@
 
 public class HtmlParameter implements Comparable<HtmlParameter> {
 	public enum Type {
-		cookie, form, url
+		cookie, form, url, header
```

Comment:
OK, lets leave it all lower case then (if/when we break compatibility we can change all of them).